### PR TITLE
[13.0][IMP] stock_picking_batch_extended_account: Add some tests. And more inheritable

### PR DESCRIPTION
**Pull request by sergio-teruel**
cc @Tecnativa TT36237

ping @carlosdauden @ernestotejeda

### CODE REVIEW

**Review by carlosdauden — APPROVED** [review]

**Review by ernestotejeda — APPROVED** [review]
Code and functional review: :+1:

File: stock_picking_batch_extended_account/migrations/13.0.1.1.0/post-migration.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+from openupgradelib import openupgrade  # pylint: disable=W7936
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    openupgrade.logged_query(
+        env.cr,
+        """
+        UPDATE ir_property
+        SET value_text = CASE WHEN value_integer = 1 THEN 'yes' ELSE 'no' END,
```
Comment (pedrobaeza) [inline stock_picking_batch_extended_account/migrations/13.0.1.1.0/post-migration.py:16]: Or is it `0`?

```suggestion
        SET value_text = CASE WHEN value_integer = 1 THEN 'yes' ELSE 'no' END,
            value_integer=NULL,
```

File: stock_picking_batch_extended_account/migrations/13.0.1.1.0/post-migration.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+        type='char'
+        WHERE name='batch_picking_auto_invoice'
```
Comment (pedrobaeza) [inline stock_picking_batch_extended_account/migrations/13.0.1.1.0/post-migration.py]: Please link by a stronger condition, as there can be another property field with the same name.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

**Comment by sergio-teruel** [comment]
@pedrobaeza Changes done!!

File: stock_picking_batch_extended_account/migrations/13.0.1.1.0/post-migration.py (hunk 2 above)
Comment (pedrobaeza) [reply stock_picking_batch_extended_account/migrations/13.0.1.1.0/post-migration.py]: Well, this is still vague. Please use `fields_id`.

Comment (sergio-teruel) [reply stock_picking_batch_extended_account/migrations/13.0.1.1.0/post-migration.py]: Done!

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**